pkg/upgrade/releases.go

```diff
@@ -19,9 +19,7 @@ const (
 )
 
 // LatestRelease retrieves latest release information for given releaseStream. Will use Cincinnati for stage/prod.
-func LatestRelease(cfg *config.Config) (name, pullSpec string, err error) {
-	releaseStream := cfg.UpgradeReleaseStream
-
+func LatestRelease(cfg *config.Config, releaseStream string) (name, pullSpec string, err error) {
```

Comment:
Nit: Do we need the releaseStream argument since it's part of config anyway?

Reply:
Yes. `LatestRelease` looks for the latest valid release in a specific stream. In some cases we pass `cfg.UpgradeReleaseStream` and in some we pass `cfg.TargetStream`

We also pass the `cfg` var because LatestRelease will update it with version info based on the stream passed.

Reply:
SGTM. Will resolve!